faucet/acl.py

```diff
@@ -435,3 +435,10 @@ def resolve_ports(self, resolve_port_cb, resolve_tunnel_objects):
         'output': {'set_fields': [
             {'eth_src': '01:02:03:04:05:06'}, {'eth_dst': '01:02:03:04:05:06'}]}}}]})
 PORT_ACL_8021X.build({}, None, 1)
+MAB_ACL_8021X = ACL(
+    'mab_acl_8021x', 0,
+    {'rules': [{'eth_type': 0x0800, 'eth_src': '01:02:03:04:05:06', 'ip_proto': 0x11,
```

Comment:
Take ip_proto and eth_type from https://github.com/faucetsdn/faucet/blob/bda73db3f2189d44d17b5e4be08c4de075c13e43/faucet/valve_packet.py#L483-L488